src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User user = new User();
+    private static final User actual = new User();
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user.setLogin("FirstPerson");
+        user.setPassword("thebestpassword");
+        user.setAge(20);
+        storageDao.add(user);
+    }
+
+    @BeforeEach
+    void setup() {
+        actual.setLogin("SecondPerson");
+        actual.setLogin("anotherpass");
+        actual.setAge(36);
+    }
+
+    @Test
+    void register_null_pas_NotOk() {
```

Comment:
```suggestion
    void register_nullPassword_NotOk() {
```

**Same for other methods**